PIconnect/PI.py

```diff
@@ -110,6 +115,11 @@ def __init__(
             self._credentials = None
         self.connection = self.servers.get(server, self.default_server)
 
+        if timeout:
+            from System import TimeSpan
+
+            self.connection.ConnectionInfo.OperationTimeOut = TimeSpan(0, 0, timeout)  # hour, min, sec
```

Comment:
The black formatting tool thinks this line is too long, perhaps we can integrate the comment in the code by changing it to something like this:

```python
hours, minutes, seconds = 0, 0, timeout
self.connection.ConnectionInfo.OperationTimeOut = TimeSpan(hours, minutes, seconds)
```

Please run `black PIconnect` from the root of the project afterwards to make sure it is within the currently configured
line length.

Reply:
Hola, @Hugovdberg,
not sure that it is normal to "create" more args than we need, e.g.
```
def foo():
    a, b = 0, 0
    return a, b

def not_a_foo():
    return 0, 0
```

I believe such one-time used arguments do not require vars, - only comment. So I relocate the comment